File: 백준/Bronze/4153. 직각삼각형/직각삼각형.java
import java.util.Scanner;
import java.util.Arrays;

// 2023.11.26 13:46

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        while (true) {
            int a = sc.nextInt();
            int b = sc.nextInt();
            int c = sc.nextInt();

            if (a == 0) break;

            boolean tri = rightTri(a, b, c);

            if (tri) System.out.println("right");
            else System.out.println("wrong");
        }

        sc.close();
    }

    static boolean rightTri(int a, int b, int c) {
        return (((a*a) == (b*b) + (c*c)) || ((b*b) == (a*a) + (c*c)) || ((c*c) == (b*b) + (a*a)));
    }
}

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.js
const input = require('fs')
  .readFileSync(process.platform === "linux" ? '/dev/stdin' : './input.txt')
  .toString()
  .trim()
  .split("\n")
  .map((el) => el.split(" ").map(Number));


for (let i = 0; i < input.length; i++) {
  let tri = input[i].sort((a, b) => a - b);

  if (tri[0] === 0) break;

  if (tri[0] ** 2 + tri[1] ** 2 === tri[2] ** 2) {
    console.log("right");
  } else {
    console.log("wrong")
  }
}